lld/test/ELF/aarch64-pauth-rela-iplt-end.s

```diff
@@ -0,0 +1,20 @@
+# REQUIRES: aarch64
+# RUN: llvm-mc -filetype=obj -triple=aarch64 %s -o %t.o
+# RUN: ld.lld --static %t.o -o %t
```

Comment:
`-static` is redundant (unlike the compiler driver `-static`, our `-static` just disables searching for / linking against DSO)